reference/tools.rst

```diff
@@ -478,13 +478,18 @@ Parameters:
 tools.pythonpath()
 ------------------
 
+This tool is automatically applied in the conanfile methods unless :ref:`apply_env<apply_env>` is deactivated, so
+any PYTHONPATH inherited from the requirements will be automatically available.
+
 .. code-block:: python
 
     def pythonpath(conanfile)
 
 This is a context manager that allows to load the PYTHONPATH for dependent packages, create packages
 with python code, and reuse that code into your own recipes.
 
+It is automatically applied
+
 .. code-block:: python
 
     from conans import tools
```

Comment:
If I understood this feature well, using `tools.pythonpath()` is no longer needed if `apply_env `is `True`. So this code

```
def build(self):
    with tools.pythonpath(self):
        from module_name import whatever
        whatever.do_something()
```
can still be used even if `apply_env` is `True` and would be the same as typing:

```
def build(self):
    from module_name import whatever
    whatever.do_something()
```
is that right?

Reply:
True. Do you want me to add these comments?